index.html

```diff
@@ -202,71 +203,82 @@ <h3>Profesional Art Printing Data</h3>
-        </p>
-        <button class="btn-1">Get My Resume</button>
-      </div>
-    </div>
-
-    <div class="card">
-      <div class="card-experience">
-        <section>
-          <h4 class="dark"> <span class="figure-1"></span> Language </h4>
-          <ul class="skill-list dark">
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        </section>
-
-        <section>
-          <h4 class="dark"> <span class="figure-2"></span> Frameworks </h4>
-          <ul class="skill-list dark">
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </section>
+  <section class="about-me-intro">
+    <header>
+      <h2>
+        About me
+      </h2>
+    </header>
+    <p class="dark">Hello I’m a software developer! I can help you build a product , feature or website Look
+      through some of my work and experience! If you like what you see and have
+      a project you need coded, don’t hestiate to contact me.
+    </p>
+    <button class="btn-1">Get My Resume</button>
+  </section>
 
-        <section>
-          <h4 class="dark"> <span class="figure-3"></span> Skills </h4>
-          <ul class="skill-list dark">
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </section>
+  <section class="skill-techs">
+    <h4 class="dark"> <span class="figure-1"></span> Language </h4>
+    <ul class="skill-list dark">
+      <li>JavaScript</li>
+      <li>Ruby</li>
+      <li>Html</li>
+      <li>Css</li>
+    </ul>
 
-      </div>
-    </div>
+    <h4 class="dark"> <span class="figure-2"></span> Frameworks </h4>
+    <ul class="skill-list dark">
+      <li>Bootstrap</li>
+      <li>Ruby on Rails</li>
+      <li>RSPec</li>
+      <li>CapyBara</li>
+      <li>Selenium</li>
+    </ul>
 
-    <div class="card">
+    <h4 class="dark"> <span class="figure-3"></span> Skills </h4>
+    <ul class="skill-list dark">
+      <li>Codekit</li>
+      <li>Github</li>
+      <li>Codepen</li>
+      <li>Gitlab</li>
+      <li>Terminal</li>
+    </ul>
+  </section>
 
-    </div>
+  <section class="contact-me">
+    <header>
+      <h3>
+        I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.
+      </h3>
+    </header>
+    <form action="https://formspree.io/f/mbjqyrqj" method="POST">
+      <fieldset>
+        <input oninvalid="setCustomValidity('Please enter your full name')" type="text" name="name" id="fullname"
+          required maxlength="30" placeholder="Full name">
+        <input type="email" name="_replyto" oninvalid="setCustomValidity('Please enter a valid email')" id="email"
+          required placeholder="Email address">
```

Comment:
The form is not submitting even when all fields are properly filled this is what it looks like 
![Screenshot (21)](https://user-images.githubusercontent.com/52497246/121265509-d3508180-c8b0-11eb-83d2-d6c79a3767f3.png)

Reply:
That's odd
it's working just fine with me 
![Screenshot_2021-06-09-00-32-27-18](https://user-images.githubusercontent.com/47625407/121266139-4ad6de80-c8ba-11eb-83e9-9a2d3dc48db5.jpg)
![Screenshot_2021-06-09-00-32-18-69](https://user-images.githubusercontent.com/47625407/121266146-4ca0a200-c8ba-11eb-8502-5cc47cd09ddc.jpg)

Reply:
Yes it was also working fine with me but just now that I checked it is no longer working

Reply:
It's still valid for me even now.

Reply:
Now its working again 
![image](https://user-images.githubusercontent.com/28903404/121266551-4ab9ed00-c880-11eb-8f9c-0ae2f2495315.png)
![image](https://user-images.githubusercontent.com/28903404/121266564-50173780-c880-11eb-8ed8-4841127757ed.png)

Reply:
What should I do? :fearful:

Reply:
https://formspree.io/f/myylerlb

Try this one instead of yours and test it locally, then let me know here please

Reply:
It works 👍🏻

Reply:
> https://formspree.io/f/myylerlb
> 
> Try this one instead of yours and test it locally, then let me know here please

Done, I think it went thru no errors on my machine

Reply:
> It works 👍🏻

should I merge? or is there a way to know what caused that?

Reply:
I guess it stopped working for you on the your link because you spammed emailing while you're testing it. Well, now we know you have nothing to worry about and you're good to go.
submit another CRR now. 
good luck and happy coding with the next.

Reply:
Should I submit it with the link to my formspree or with yours?

Reply:
it doesn't matter, but use yours. Anyways, the next reviewer will understand the situation from this conversation.

Reply:
> it doesn't matter, but use yours. Anyways, the next reviewer will understand the situation from this conversation.

hope he does, its my last review :fearful:

Reply:
It works on my end now 👍